packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -96,7 +98,7 @@ export interface CommonCallingHandlers {
 /**
  * options bag to start captions
  *
- * @beta
+ * @public
```

Comment:
There is a lot of caption related changes here, they are for captions GA and the corresponding calling version is going to be release on 8/21. The changes should not be here

Reply:
Good catch. Will change this back to beta